Patches/Forsakens/Forsakens_CE_Patch_ThingDefs_Weapons.xml

```diff
@@ -0,0 +1,561 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Forsakens</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<!-- ========== Forsaken Sniping Rifle ========== -->
+
+				<!-- (CE implementation based on Accuracy International AWM sniper rifle from RN Weapons, chambered for 8x35mm Charged cartridge) -->
+				<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+					<defName>ForsakenSniper</defName>
+					<statBases>
+						<Mass>7.18</Mass>
+						<RangedWeapon_Cooldown>1.37</RangedWeapon_Cooldown>
+						<SightsEfficiency>3.50</SightsEfficiency>
+						<ShotSpread>0.01</ShotSpread>
+						<SwayFactor>1.61</SwayFactor>
+						<Bulk>15.60</Bulk>
+						<WorkToMake>50500</WorkToMake>
+					</statBases>
+					<costList>
+						<!-- from original mod -->
+						<Steel>300</Steel>
+						<Plasteel>75</Plasteel>
+						<ComponentIndustrial>12</ComponentIndustrial>
+					</costList>
+					<Properties>
+						<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+						<hasStandardCommand>true</hasStandardCommand>
+						<defaultProjectile>Bullet_8x35mmCharged_ForsakenSniper</defaultProjectile>
+						<warmupTime>2.55</warmupTime>
+						<range>107</range>
```

Comment:
Is there a compelling reason the range needs to be this high? While I understand that the AWM has an effective range of over 1000m, that's really sort of the maximum used to the generate stats for weapons. The 90mm flak and the Tac-50 have range of 86, so this would put its range markedly above any other direct-fire weapon.

Reply:
No good reason for it to be that high, so I'm dropping it to 86.

Fixed in 95571ae